client.go

```diff
@@ -30,3 +41,28 @@ func NewClient(c *Config) *Client {
 	}
 	return client
 }
+
+//waitForRequestCompleted allows to wait for a request to complete. Timeouts are currently hardcoded
```

Comment:
`Timeouts are currently hardcoded` is this statement stil valid?

Reply:
I'll fix.